specs/experimental/op-contracts-manager.md

```diff
@@ -280,36 +271,45 @@ The `addGameType` method is used to orchestrate the actions required to add a ne
 or more chains.
 
 ```solidity
-struct PermissionlessGameConfig {
-  bytes32 absolutePrestate;
+struct AddGameInput {
+    string saltMixer;
+    ISystemConfig systemConfig;
+    IProxyAdmin proxyAdmin;
+    IDelayedWETH delayedWETH;
+    GameType disputeGameType;
+    Claim disputeAbsolutePrestate;
+    uint256 disputeMaxGameDepth;
+    uint256 disputeSplitDepth;
+    Duration disputeClockExtension;
+    Duration disputeMaxClockDuration;
+    uint256 initialBond;
+    IBigStepper vm;
+    bool permissioned;
+}
+
+struct AddGameOutput {
+    IDelayedWETH delayedWETH;
+    IFaultDisputeGame faultDisputeGame;
 }
 
-function addGameType(ISystemConfig[] _systemConfigs, PermissionlessGameConfig[] _newGames) public;
+/// @notice addGameType deploys a new dispute game and links it to the DisputeGameFactory. The inputted _gameConfigs
+/// must be added in ascending GameType order.
+function addGameType(AddGameInput[] memory _gameConfigs) external returns (AddGameOutput[] memory)
 ```
```

Comment:
We forgot to add an event to the spec and the implementation. I'm just adding it now in [this PR](https://github.com/ethereum-optimism/optimism/pull/14196)